src/components/package-port/EditorNav.tsx

```diff
@@ -190,6 +190,18 @@ export default function EditorNav({
     isLoggedIn &&
       (!pkg || pkg?.owner_github_username === session?.github_username),
   )
+
+  useEffect(() => {
+    const handleKeyDown = (e: KeyboardEvent) => {
+      if ((e.ctrlKey || e.metaKey) && e.key === "s") {
+        e.preventDefault()
+        if (!hasUnsavedChanges || !canSavePackage) return
+        onSave()
+      }
+    }
+    window.addEventListener("keydown", handleKeyDown)
+    return () => window.removeEventListener("keydown", handleKeyDown)
+  }, [onSave, hasUnsavedChanges, isLoggedIn, canSavePackage])
```

Comment:
```suggestion
  }, [onSave])
```

Do we really need these?